T05: Setup example Angular app

jaca1119 opened the pull request:
Create simple Angular app which consumes translations from the server

jaca1119 committed:
T05: Simple example of translations usage

jaca1119 committed:
T05: Add top bar in example-app2

jaca1119 committed:
T05: Add categories list card in example-app2

jaca1119 committed:
T05: Add title to categories card in example-app2

ghost commented:
<img src="https://www.deepcode.ai/icons/green_check.svg" width= "50px" align= "left"/> Congratulations :tada:. DeepCode [analyzed](https://www.deepcode.ai/app/gh/tubidubidam/tim2/08ab036e92c396ed2036add27a547e2ba8180fe5/tubidubidam/tim2/15361bea867ae4fec3b06bf547605eece6819063/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=0&) your code in 2.677 seconds and we found no issues. Enjoy a moment of no bugs :sunny:.

#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/tubidubidam/tim2/08ab036e92c396ed2036add27a547e2ba8180fe5/tubidubidam/tim2/15361bea867ae4fec3b06bf547605eece6819063/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=0&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=tubidubidam)

jaca1119 committed:
T05: Fix exporting project messages for source locale in CDExportsService

jaca1119 committed:
T05: Fix overriding translations by messages in CDExportsService

jaca1119 committed:
T05: Add category list translations

soyoboyo committed:
# Conflicts:
#	backend/src/main/java/org/tim/constants/Mappings.java

jaca1119 committed:
T05: Add hot product card in example-app2

soyoboyo committed:
FIx indents in all backend files

soyoboyo committed:
# Conflicts:
#	backend/src/main/java/org/tim/constants/Mappings.java
#	backend/src/main/java/org/tim/databaseSeed/MessageSeeder.java
#	backend/src/main/java/org/tim/databaseSeed/ProjectSeeder.java
#	backend/src/main/java/org/tim/databaseSeed/TranslationSeeder.java
#	backend/src/test/java/org/tim/configuration/SpringTestsCustomExtension.java
#	frontend/src/app/app.module.ts
#	frontend/src/app/features/developers/dev-messages/dev-messages.component.ts
#	frontend/src/app/features/translators/tra-messages/tra-messages-table/tra-messages-table.component.html
#	frontend/src/app/features/translators/tra-messages/tra-messages.component.html
#	frontend/src/app/features/translators/tra-messages/tra-messages.component.ts

soyoboyo committed:
# Conflicts:
#	auth_server/src/main/java/com/oauth/server/services/AuthoritySeed.java
#	backend/src/main/java/org/tim/constants/Mappings.java
#	backend/src/main/java/org/tim/databaseSeed/MessageSeeder.java
#	backend/src/main/java/org/tim/databaseSeed/ProjectSeeder.java
#	backend/src/main/java/org/tim/databaseSeed/TranslationSeeder.java
#	backend/src/test/java/org/tim/configuration/SpringTestsCustomExtension.java

soyoboyo committed:
Merge branch 'develop' into feature/example-app

# Conflicts:
#	backend/src/main/java/org/tim/constants/Mappings.java
#	backend/src/main/java/org/tim/databaseSeed/MessageSeeder.java
#	backend/src/main/java/org/tim/databaseSeed/ProjectSeeder.java
#	backend/src/main/java/org/tim/databaseSeed/TranslationSeeder.java

jaca1119 committed:
T05: Added featured products list to example-app2. Fixed DeepCode issue

jaca1119 committed:
T05: Add footer with translations in example-app2

jaca1119 committed:
T05: Download all translations for project in ZIP

jaca1119 committed:
T05: Use translations from json files

jaca1119 committed:
T05: Use com.fasterxml.jackson.databind.ObjectMapper instead of org.codehaus.jackson.map.ObjectMapper
